src/main/java/com/gmail/yuramitryahin/thread/CallableThread.java

```diff
@@ -0,0 +1,20 @@
+package com.gmail.yuramitryahin.thread;
+
+import java.util.List;
+import java.util.concurrent.Callable;
+
+public class CallableThread implements Callable<Integer> {
```

Comment:
name classes according to their purpose
```suggestion
public class SumCalculator or SumCalculatingThread implements Callable<Integer> {
```

Reply:
fixed